src/mito/src/engine/procedure/create.rs

```diff
@@ -151,15 +151,18 @@ impl<S: StorageEngine> CreateMitoTable<S> {
             &self.data.request.schema_name,
             self.data.request.id,
         );
+        let table_options = &mut self.data.request.table_options;
```

Comment:
```suggestion
        let table_options = &self.data.request.table_options;
```

Reply:
Ah.. good spot! thanks @v0y4g3r

Reply:
> Ah.. good spot! thanks @v0y4g3r

May I just commit this change for you?

Reply:
Give me 5 minutes please, I'm about to push this adjustment

Reply:
> Give me 5 minutes please, I'm about to push this adjustment

Don't forget `&`, you still need to borrow `self.data.request.table_options`.

Reply:
sorry, my bad :(

Reply:
@v0y4g3r could you please take a look once again?